build/crds/sedna.io_federatedlearningjobs.yaml

```diff
@@ -44,6 +44,7 @@ spec:
                   should have
                 properties:
                   model:
+                    description: TrainModel defines model of federated learning job
```

Comment:
The description is `TrainModel`, but the key is model

Reply:
fix it